Amplify/Categories/DataStore/Model/PropertyPath.swift

```diff
@@ -123,4 +123,12 @@ open class ModelPath<ModelType: Model> : PropertyContainerPath {
     public func time(_ name: String) -> FieldPath<Temporal.Time> {
         FieldPath(name: name, parent: self)
     }
+    
+    public func int(_ name: String) -> FieldPath<Int> {
+        FieldPath(name: name, parent: self)
+    }
+    
+    public func double(_ name: String) -> FieldPath<Double> {
+        FieldPath(name: name, parent: self)
+    }
```

Comment:
This is the only functional change in this PR which is why it is a "fix", the newly added schemas have fields of type `int` and `double` which uses these methods. They are added to the integraton test target to ensure they compile, provisioned to the shared backend to anticipate more testing